server/app/routes.py

```diff
@@ -13,3 +19,20 @@ def videos_raw():
 	@app.route('/videos')
 	def videos():
 		return jsonify(rss.summarize_watchlist(app.config['watchlist']))
+
+	@app.route('/watchlist', methods=['POST', 'DELETE'])
+	def modify_watchlist():
+		# only required value is 'id'
+		json = request.get_json()
+		if 'id' not in json:
```

Comment:
i'm thinking the backend should actually accept channel url because we'll need a way to determine channel id for named channels and that's difficult to do on the front end due to cors restrictions.

Reply:
and in a 200 scenario it should return the channel id

Reply:
Yeah, I'm planning to add that in eventually.

Reply:
Actually, on second thought, I'm just going to make it accept only channel url.
https://www.youtube.com/channel/<channel_id> works even if it's a named channel anyways.
Better to just keep it simple with only one way to specify a channel.

Reply:
On the third thought, I'm going to add the option to only use channel url, but keep this route.
I think it would make deleting entries easier when I add a GET option for /watchlist.

Reply:
Yeah, I meant only accept channel url, no need to have options for both since front end can just generate a url if it has an id